<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseFile</class>
 <widget class="QWidget" name="ChooseFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="filepath_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>741</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>filepath:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="choose_btn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>681</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(224, 224, 224);
    border-radius: 5px;
    border: 2px solid rgb(184, 184, 184);
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(200, 200, 200);
    border: 2px solid rgb(154, 154, 154);
}</string>
   </property>
   <property name="text">
    <string>Choose file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>470</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(224, 224, 224);
    border-radius: 5px;
    border: 2px solid rgb(184, 184, 184);
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(200, 200, 200);
    border: 2px solid rgb(154, 154, 154);
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_btn">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>470</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(224, 224, 224);
    border-radius: 5px;
    border: 2px solid rgb(184, 184, 184);
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(200, 200, 200);
    border: 2px solid rgb(154, 154, 154);
}</string>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
